src/toil/wdl/wdl_synthesis.py

```diff
@@ -741,18 +745,23 @@ def write_function_dockercall(self, job):
                        "job_task_reference": job,
                        "docker_user": str(self.docker_user)}
         docker_template = heredoc_wdl('''
-        stdout = apiDockerCall(self, 
-                               image={docker_image}, 
-                               working_dir=tempDir, 
-                               parameters=[os.path.join(tempDir, "{job_task_reference}_script.sh")], 
-                               entrypoint="/bin/bash", 
-                               user={docker_user}, 
-                               stderr=True, 
-                               volumes={{tempDir: {{"bind": tempDir}}}})
-        writetype = 'wb' if isinstance(stdout, bytes) else 'w'
-        with open(os.path.join(asldijoiu23r8u34q89fho934t8u34fcurrentworkingdir, '{job_task_reference}.log'), writetype) as f:
-            f.write(stdout)
-            ''', docker_dict, indent='        ')[1:]
+        # apiDockerCall() with demux=True returns a tuple of bytes objects (stdout, stderr).
+        _toil_wdl_internal__stdout, _toil_wdl_internal__stderr = \\
+            apiDockerCall(self, 
+                          image={docker_image}, 
+                          working_dir=tempDir, 
+                          parameters=[os.path.join(tempDir, "{job_task_reference}_script.sh")], 
+                          entrypoint="/bin/bash", 
+                          user={docker_user}, 
+                          stderr=True, 
```

Comment:
@adamnovak Thanks for the review! The newly-available Docker output has been implemented here.